cmd/podman/shared/create.go

```diff
@@ -709,7 +709,7 @@ func ParseCreateOpts(ctx context.Context, c *GenericCLIResults, runtime *libpod.
 
 	// SECCOMP
 	if data != nil {
-		if value, exists := data.Annotations[seccompAnnotationKey]; exists {
+		if value, exists := labels[seccompLabelKey]; exists {
```

Comment:
Should we check both? It doesn't seem like it would hurt.

Reply:
I would prefer to have just one place, especially when aiming to get that into the OCI. Luckily, nobody is using it yet :-)